test/mjsunit/wasm/asm-wasm.js

```diff
@@ -339,3 +339,119 @@ function TestMixedAdd() {
 
 assertEquals(23, WASM.asmCompileRun(TestMixedAdd.toString()));
 */
+
+function TestConvertI32() {
+  "use asm";
+
+  function caller() {
+    var a = 1.5;
+    if ((~~(a + a)) == 3) {
+      return 24;
+    }
+    return 0;
+  }
+
+  return {caller:caller};
+}
+
+assertEquals(24, WASM.asmCompileRun(TestConvertI32.toString()));
+
+function TestConvertF64() {
+  "use asm";
+
+  function caller() {
+    var a = 1;
+    if ((+(a + a)) > 1.5) {
+      return 25;
+    }
+    return 0;
+  }
+
+  return {caller:caller};
+}
+
+assertEquals(25, WASM.asmCompileRun(TestConvertF64.toString()));
+
+function TestConvertF64FromUnsigned() {
```

Comment:
We should probably chat about whether we want to consistantly start using the asm.js type names. May or may not make sense everywhere.